`Zim-tools` version `3.3.0` splitted zim files giving `Corrupted block detected` error when there is a 0 size extra ZIM chunk.

**Describe the bug**
* For full details please see https://github.com/kiwix/kiwix-android/pull/3803. 
* But the main issue that we are facing is the `Corrupted block detected` error when getting the `getArticleCount()` and `getMediaCount()` with zim-tools's splitted zim files(When there is a 0size size extra part) However, we can read the zim file content. If we delete that 0 size extra part from our storage then there is no error and it starts giving the value from these methods.
* We have written the test cases in https://github.com/kiwix/kiwix-android/pull/3803 for testing this scenario where the 0 size part is present in the storage, but the test is passing because, with that zim file, there is no error in getting the `getArticleCount()` and `getMediaCount()`. Also, if we split any zim file with this test case and tries to load it in our application then these methods retuns the value.
* One more issue we have detected of this type of error, when we tries to save the bookmark then the application crashes with these logs, when we tries to use the `update(Archive archive)` method of the `Book` class.

```kotlin
 Process: org.kiwix.kiwixmobile, PID: 17268
                                                                                                    java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:562)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
                                                                                                    Caused by: java.lang.reflect.InvocationTargetException
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878) 
                                                                                                    Caused by: java.lang.Exception: Corrupted block detected
                                                                                                    	at org.kiwix.libkiwix.Book.update(Native Method)
                                                                                                    	at org.kiwix.kiwixmobile.core.main.CoreReaderFragment.toggleBookmark(CoreReaderFragment.kt:1758)
                                                                                                    	at org.kiwix.kiwixmobile.core.main.CoreReaderFragment_ViewBinding$6.doClick(CoreReaderFragment_ViewBinding.java:121)
                                                                                                    	at butterknife.internal.DebouncingOnClickListener.onClick(DebouncingOnClickListener.java:26)
                                                                                                    	at android.view.View.performClick(View.java:7729)
                                                                                                    	at android.view.View.performClickInternal(View.java:7706)
                                                                                                    	at android.view.View.-$$Nest$mperformClickInternal(Unknown Source:0)
                                                                                                    	at android.view.View$PerformClick.run(View.java:30484)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:959)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:100)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:232)
                                                                                                    	at android.os.Looper.loop(Looper.java:317)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8501)
```

**Expected behavior**
It should return the value from the `getArticleCount()` and `getMediaCount()` methods with `zim-tools` splitted zim file as well.

--- a/buildSrc/src/main/kotlin/Versions.kt
+++ b/buildSrc/src/main/kotlin/Versions.kt
@@ -84,7 +84,7 @@ object Versions {
 
   const val core_ktx: String = "1.9.0"
 
-  const val libkiwix: String = "2.2.1"
+  const val libkiwix: String = "2.2.2"
 
   const val material: String = "1.8.0"
 

